include/core/clipboard.hpp

```diff
@@ -40,8 +41,9 @@ namespace Toolbox {
 
         Result<std::string, ClipboardError> getText() const;
         Result<void, ClipboardError> setText(const std::string &text);
-        Result<MimeData, ClipboardError> getContent(const std::string &type) const;
-        Result<void, ClipboardError> setContent(const std::string &type, const MimeData &mimedata);
+        Result<MimeData, ClipboardError> getContent() const;\
```

Comment:
Trailing `\` ?

Reply:
Fixed in [this commit](https://github.com/JoshuaMKW/JuniorsToolbox/pull/36/commits/f58e53b35b3af6140ee16491b626989acb307ba6)